cli/cmd/generate_gcp.go

```diff
@@ -54,6 +52,9 @@ var (
 	QuestionGcpAnotherAdvancedOpt      = "Configure another advanced integration option"
 	GcpAdvancedOptLocation             = "Customize output location"
 	QuestionGcpCustomizeOutputLocation = "Provide the location for the output to be written:"
+	QuestionGcpCustomFilter            = "Specify a custom Audit Log filter which will supersede all other filter options"
```

Comment:
A nit, but just to put it in present tense, which I think is preferable here....

```suggestion
	QuestionGcpCustomFilter            = "Specify a custom Audit Log filter which supersedes all other filter options"
```